### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
## What does this PR do
In this PR me and my coding partner check the error about the popup and make the changes.
## Description of the task to be completed
- [x] Reformated work section and created it dynamically using javascript
- [x] Created a modal containing dynamic data for each card

## How should it be tested
Clone the repository
```
git clone https://github.com/AhmedMohsen600/portfolio-setup-mobile-first.git
```
Navigate to the folder
```
cd portfolio-setup-mobile-first
```
git checkout popup
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @AhmedMohsen600,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Linters are green ✅.
- PR is very well made 👍.
- The readme file is nice.


## Required Changes ♻️
_Check the comments under the review._

- Hello and nice work on the poop-up but the modal is too big when it's opened please re-adjust the modal size ( width, and height), and also the buttons are not symmetric as the provided design,  and also please add the repo link into the readme file so the reader can see it and copy it.

Capture web_12-10-2022_134731_localhost


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Github flow 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 1]
```diff
@@ -19,3 +34,196 @@ const handelLinkClick = () => {
...
+    modalSkills: new Set([
+      'html',
+      'css',
+      'javascript',
+      'github',
+      'ruby',
+      'Bootstrap',
+    ]),
+    title: 'Multi-Post Stories',
+    canopy: 'CANOPY',
```
[the remark this answers is not in the document]
Comment (AhmedMohsen600) [reply app.js:74]: @Davidosky007 Thank you I did fix it and push again

[the remark this answers is not in the document]
Comment (Davidosky007) [reply app.js:74]: Thank you

**Pull request merged**